CHANGELOG.md

```diff
@@ -30,6 +30,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Eric Amorde](https://github.com/amorde)
   [#12122](https://github.com/CocoaPods/CocoaPods/issues/12122)
 
+* Support `visionos` in `pod lib lint --platforms=` and use `xros` for `Fourflusher`  
```

Comment:
Nit: this is in the wrong spot since 1.14.3 shipped. Also you can use this PR number instaed of #issue_number

Reply:
Oops, my apologies! I will move it 😊